isofit/radiative_transfer/engines/modtran.py

```diff
@@ -313,6 +313,10 @@ def makeSim(self, point, file=None, timeout=None):
         vals["FILTNM"] = os.path.normpath(self.filtpath)
         modtran_config_str, modtran_config = self.modtran_driver(dict(vals))
 
+        # Translate to the MODTRAN OBSZEN convention
+        if vals["OBSZEN"]<90:
+            vals["OBSZEN"] = 180 - abs(vals["OBSZEN"])
+
```

Comment:
@davidraythompson I think this needs to be implemented before line 314, so that the modtran_driver() function receives the correct OBSZEN. After line 314, the `vals` dictionary is not used anymore. Let me know if you want me to change this and I'm happy to push directly to your branch.

Reply:
Fixed, thanks :)